docs/blog/Webots-2021-b-release.md

```diff
@@ -0,0 +1,111 @@
+
+---
+
+## New Web Rendering Engine
+
+Since the introduction of the web interface for streaming simulations and playing recorded animations, we struggled finding optimal parameters for shadows and illuminations that will reproduce exactly the same rendering as in the Webots Desktop application.
+Finally, we decided to port our custom 3D renderer WREN to WebAssembly and use it for the web simulation interface in order to minimize the differences with the Desktop version and make it straightforward to run high quality simulations and animations on the web.
+
+During this work, we also took the time to refresh and improve the interface.
+
+%figure "New Wen Simulation Interface"
+![Web Simulation Interface](images/web_simulation_interface.png)
+%end
+
+---
+
+## Extended Motor and Joint Functionalities
+
+The [Motor](../reference/motor.md) and [Joint](../reference/joint.md) nodes have been extended to provide more realistic simulations.
+
+We enhanced the `Motor` node to support coupled motors specifying the linked motor in the `name` field and setting their ratio using the new `multiplier` field.
+
+Then, we added two new PROTO nodes [HingeJointWithBacklash](../guide/hinge-joint-with-backlash) and [Hinge2JointWithBacklash](../guide/hinge-2-joint-with-backlash) to easily add backlash effect and a [Gear](..//doc/guide/object-gear.md) model to simulate collision-based transmission.
+
+TODO add figure
+
+---
+
+## New Robot Models
+
+We've added many new robot models in this release.
+
+### Modular Mobile Platforms
+
+We just added three new models of modular mobile platform robots used for logistic and indoor transport.
+All these models have predefined fields to mount additional devices on the platform and front/back lidars.
+
+TODO show the three robot figures on a single line
+
+#### [Robotnik Summit-XL Steel](../guide/summit-xl-steel.md)
+%figure "Summit-XL Steel in Webots"
+![Summit-XL Steel in Webots](images/summit_xl_steel.wbt.thumbnail.jpg)
+%end
+
+#### [Mobile Industrial Robots MiR100](../guide/mir100.md)
+%figure "MiR100 in Webots"
+![MiR100 in Webots](images/mir100.wbt.thumbnail.jpg)
+%end
+
+#### [REC Fabtino](../guide/fabtino.md)
+
+%figure "Fabtino in Webots"
+![Fabtino in Webots](images/fabtino.wbt.thumbnail.jpg)
+%end
+
+### NVIDIA JetBot
+
+The NVIDIA [JetBot](../guide/jetbot.md) is an affordable two-wheeled robot based on the NVIDIA Jetson Nano board widely used in AI robotics applications.
+
+%figure "JetBot in Webots"
+![JetBot in Webots](images/jetbot.wbt.thumbnail.jpg)
+%end
+
+A standalone model of the NVIDIA [Jetson Nano](../guide/single-board-computers.md#nvidia-jetson-nano) board is also available.
+
+---
+
+## ROS 2 & Webots
+
+TODO
+
+---
+
+## Extra Goodies
+
+A new `Billboard` node has been introduced to display information in the main view regardless of the positoin and orientation of the viewpoint.
+
+The [SICK S300](../guide/lidar-sensors.md#sick-s300) lidar model is now included in our sensors library.
```

Comment:
A picture of the SICK S300 would be nice here.

Reply:
Usually we never added pictures in the extra goodies section. 
If we want to add a picture, then it is better to create a new section for new sensors.

Reply:
Then, I believe it deserves a new sensors section.